### PR TITLE
Orca: MMCV Training Example on fast_rcnn

### PR DESCRIPTION
**Pull request by leonardozcm**
## Description

- Add a mmcv faster-rcnn training example with orca
- Add readme
### 1. Why the change?

related issue: https://github.com/intel-analytics/BigDL/issues/7774
### 2. User API changes
N/A

### 3. Summary of the change 

Add a mmcv faster-rcnn training example with orca and related doc.
### 4. How to test?
- [ ] N/A
- [x] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

https://github.com/intel-analytics/BigDL/pull/7846

### CODE REVIEW

File: python/orca/example/learn/mmcv/faster_rcnn/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+`--config`: config file local path on driver node, here we use the config file from mmdet repo
+
+`--load_from`: the pre-trained model path on HDFS
+
+`$MMDET_PATH`: your mmdetection repo local path on driver node.
+
+
+## Results
+
+**For "ray" backend**
```
Comment (hkvision) [inline python/orca/example/learn/mmcv/faster_rcnn/README.md:82]: remove this

File: python/orca/example/learn/mmcv/faster_rcnn/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+`--config`: the local config file path, here we use the config file from mmdet repo
+
+`--load_from`: the local pre-trained model path
+
+`$MMDET_PATH`: your mmdetection repo local path.
+
+
+- yarn-client mode
+
+To train with yarn-client mode, you need to make sure the dataset is under the same path of every worker node. In order to do this, you can upload the dataset to HDFS and download dataset to every worker node before training.
```
Comment (hkvision) [inline python/orca/example/learn/mmcv/faster_rcnn/README.md:59]: This is inappropriate.

**Review by hkvision — APPROVED** [review]
Others LGTM

File: .github/actions/orca/orca-exampletest-ray-py38-spark3-action/pr-test/action.yml [hunk 3]
```diff
@@ -33,6 +33,8 @@ runs:
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} pyarrow==4.0.1
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} albumentations
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} grpcio==1.43.0
+        pip install mmcv-full==1.7.0 -f https://download.openmmlab.com/mmcv/dist/cpu/torch1.7/index.html
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} mmdet==2.28.0
```
Comment (hamham223) [inline .github/actions/orca/orca-exampletest-ray-py38-spark3-action/pr-test/action.yml:37]: also need to install packages in `orca-exampletest-ray-py39-spark3-action`

**Pull request merged**